src/components/AddComment.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+import { postComment } from "../api";
+
+const AddComment = ({comments, review_id}) => {
+    const [commentAuthor, setCommentAuthor] = useState('');
+    const [commentBody, setCommentBody] = useState('');
+    const [commentVotes, setCommentVotes] = useState(0);
+    const [commentPosted, setCommentPosted] = useState(false)
+
+    return <section>
+        <form className="add-comment-form" onSubmit={(event) => {
+            event.preventDefault()
+
+            const commentToAdd = {
+                "author": commentAuthor,
+                "body": commentBody,
+                "votes": commentVotes
+            };
+            setCommentPosted(true)
+            postComment(review_id, commentToAdd)
+            .then((newComment) => {
+                comments((currentComments) => {
+                    return [newComment, ...currentComments];
+                })
+            })
+            .then(() => {
+                setCommentAuthor('');
+                setCommentBody('');
+                setCommentVotes(0);
+            })
+        }}>
+
+        <label htmlFor="comment-author">Comment Author</label>
+        <input name="comment-author" value={commentAuthor} required onChange={(event) =>{
+            setCommentAuthor(event.target.value)
+        }}></input>
+        <label htmlFor="comment-body">Comment</label>
+        <input name="comment-body" value={commentBody} required onChange={(event) =>{
```

Comment:
This input should be changed to allow for multiline commenting. Have a look at changing the type attribute to something other than 'text'